### PR TITLE
Feature/core 950 feedback

### PR DESCRIPTION
**Pull request by Tschuck**
- implement feedback from sde / twi

### CODE REVIEW

File: core/digital-twin-lib/src/DAppTwin.ts [hunk 1]
```diff
@@ -246,6 +236,21 @@ class DAppTwin extends DigitalTwin {
   public async setDescription(description?: DBCPDescriptionInterface): Promise<void> {
     return this.baseSetDescription(description);
   }
+
+  /**
+   * Creates a new dapp container instances, initialize it and applies it to the container
```
Comment (karladler) [inline core/digital-twin-lib/src/DAppTwin.ts]:
```suggestion
   * Creates a new dapp container instance, initializes it and applies it to the container
```

File: dapps/digital-twin-detail/src/components/DigitalTwinDetail.ts [hunk 2]
```diff
@@ -103,6 +103,16 @@ export default class DigitalTwinDetailComponent extends mixins(EvanComponent) {
 
         // initialize a new twin, but keep old reference until twin is loaded
         const newTwin = new DAppTwin(this, this.getRuntime(), this.$route.params.twin);
+
+        // if container view should be loaded initially, just load it directly
+        if (this.$route.params.container) {
+          this.$store.state.containerPreloading = Promise.resolve().then(async () => {
```
Comment (karladler) [inline dapps/digital-twin-detail/src/components/DigitalTwinDetail.ts]: `Promise.resolve().then( ` wat? 🦆

File: core/digital-twin-lib/src/DAppTwin.ts [hunk 3]
```diff
@@ -92,25 +95,19 @@ class DAppTwin extends DigitalTwin {
...
-
-      // load description, contractAddress, dispatcherStates
-      await this.containers[key].initialize();
-
-      // make contracts also directly accessible
-      this.containerContracts[this.containers[key].contractAddress] = this.containers[key];
+    await Promise.all(Object.keys(twinEntries).map(async (key: string) => {
+      const address = await twinEntries[key].value.getContractAddress();
+
+      // only load container, if it wasn't initialized before (e.g. when it was initially loaded)
```
Comment (karladler) [inline core/digital-twin-lib/src/DAppTwin.ts]: What is meant by "initialized" here? According to the `setupContainer` method it's possible that this.containerContracts[address] is set but, `container.initialize` is not called.

File: dapps/assets/src/components/DataContainer.ts [hunk 4]
```diff
@@ -47,12 +49,41 @@ export default class DataContainerComponent extends mixins(EvanComponent) {
...
+            new bcc.Profile({
+              accountId: runtime.activeAccount,
+              profileOwner: result.owner,
+              ...(runtime as bcc.ProfileOptions),
+            }),
+          );
+        }
+
+        // eslint-disable-next-line no-param-reassign
+        result.owner = await this.aliasMapping[result.owner];
```
Comment (karladler) [inline dapps/assets/src/components/DataContainer.ts]: param reassign is by a reason discouraged from airbnb ;) 
maybe you can just return map of owners assign it from the function. Better function name would than also be getOwnerMapping e.g. ...

File: dapps/digital-twin-detail/src/components/container/Entry.ts [hunk 5]
```diff
@@ -32,22 +32,20 @@ export default class ContainerEntryComponent extends mixins(EvanComponent) {
...
-  value: any;
-
   created(): void {
-    const { dispatcherData, plugin, entries } = this.$store.state.container;
+    const { plugin } = this.$store.state.container;
     this.entrySchema = plugin.template.properties[this.name].dataSchema;
-    this.value = dispatcherData[this.name] || entries[this.name];
+  }
+
+  getValue(): any {
```
Comment (karladler) [inline dapps/digital-twin-detail/src/components/container/Entry.ts]: use getter here? So the vue template doesn't need to be changed. `get value() { ... }`

File: core/digital-twin-lib/src/DAppTwin.ts (hunk 3 above)
Comment (Tschuck) [reply core/digital-twin-lib/src/DAppTwin.ts]: Its a misleading comment. Is connected to the `loadBaseInfo` function and if no `DAppContainer` instance was added to the twin.

File: dapps/assets/src/components/DataContainer.ts (hunk 4 above)
Comment (Tschuck) [reply dapps/assets/src/components/DataContainer.ts]: Hmmm... It's just for using it twice. When i rework it into the getOwnerMapping function, i need to loop over this one, to apply it to the original one. In the context of this component, it makes totally sense, to just use this function using param reassign.

**Pull request merged**